.drone.yml

```diff
@@ -0,0 +1,77 @@
+kind: pipeline
+name: test
+
+steps:
+- name: flake8
+  image: alpine/flake8:3.7.7
+  commands:
+  - flake8
+
+---
+
+kind: pipeline
+name: publish to docker hub
+
+trigger:
+  status:
+  - success
+  event:
+  - push
+  - tag
+
+depends_on:
+  - test
+
+steps:
+- name: docker push branch ${DRONE_COMMIT_BRANCH}
+  image: plugins/docker
+  settings:
+    repo: liquidinvestigations/crji-avere-europarl
+    tags: ${DRONE_COMMIT_BRANCH}
+    username:
+      from_secret: docker_username
+    password:
+      from_secret: docker_password
+  when:
+    event:
+    - push
+
+- name: docker push autotag + latest
+  image: plugins/docker
+  settings:
+    repo: liquidinvestigations/crji-avere-europarl
+    auto_tag: true
+    username:
+      from_secret: docker_username
+    password:
+      from_secret: docker_password
+
+---
+
+kind: pipeline
+name: test app
+
+depends_on:
```

Comment:
Decide on an indent style! The first part of the file has
```yaml
list:
- item
```

And the code you introduced here has:
```yaml
list:
  - item
```

They are equivalent and you can interchange them like you just did, but it's not looking too nice.